src/main/java/dasniko/testcontainers/keycloak/ExtendableKeycloakContainer.java

```diff
@@ -250,13 +249,23 @@ protected void configure() {
 
         if (customCommandParts != null) {
             logger().warn("You are using custom command parts. " +
-                "Container behavior and configuration may be corrupted. " +
-                "You are self responsible for proper behavior and functionality!\n" +
-                "CustomCommandParts: {}", customCommandParts);
+                    "Container behavior and configuration may be corrupted. " +
+                    "You are self responsible for proper behavior and functionality!\n" +
+                    "CustomCommandParts: {}", customCommandParts);
             commandParts.addAll(customCommandParts);
         }
 
-        setCommand(commandParts.toArray(new String[0]));
+        setCommand(postProcessCommandParts(commandParts).toArray(new String[0]));
+    }
+
+    /**
+     * Use this if you need to manually adjust the Keycloak command before startup.
+     *
+     * @param commandParts the generated command parts
+     * @return the adjusted command parts
+     */
+    protected List<String> postProcessCommandParts(final List<String> commandParts) {
```

Comment:
I don't like this option to be available in this container. This will lead to unexpected behavior of the testcontainer if not properly used. Thus, people will open issues and discussions, because they don't know what they doing.
If someone really needs something like this, he/she should extend the container on his/her own. That's why it is extendable.

Reply:
copy that, this was to make it more extendable but it might be error prone, removed